lib/cc/cli/path_fingerprint.rb

```diff
@@ -0,0 +1,33 @@
+require "digest/md5"
+
+module CC
```

Comment:
Using accessors on the `Issue` sounds fine to me. The just pass the issue to the fingerprinting class?

```
SourceFingerprint.new(self).compute
```